components/eam/cime_config/testdefs/testmods_dirs/eam/mmf_max_subcycle/shell_commands

```diff
@@ -0,0 +1 @@
+./xmlchange --append -id CAM_CONFIG_OPTS -val " -cppdefs ' -DMMF_DIR_NS -DMMF_MAX_SUBCYCLE ' "
```

Comment:
I wish we had a better way of specifying the NS orientation. If we were to change from that default at some point in the compset definitions, this testmod would override it.

Reply:
Yea, I agree. We could bake it into the code or make it a namelist variable instead.
making it a namelist variable would make a lot of sense. I can make a new PR for that.